tooling/nargo_fmt/src/formatter/use_tree_merge.rs

```diff
@@ -268,6 +268,11 @@ fn merge_imports_in_tree(imports: Vec<UseTree>, mut tree: &mut ImportTree) {
             PathKind::Super => tree.insert(Segment::Super),
             PathKind::Dep => tree.insert(Segment::Dep),
             PathKind::Plain => &mut tree,
+            PathKind::Resolved(_) => {
+                // TODO: Don't know whether to insert $crate or <crate name> here, or what the crate name is
+                //       Probably better to guess instead of panicking in the LS though.
```

Comment:
This case is probably never hit as the formatter runs on a parsed AST but without doing semantic analysis. Then I think `Resolved` will never be created.

Reply:
Sounds good, I'll edit the comment. Do you think we should still avoid a panic or would it be better to have a panic to verify these assumptions?

Reply:
I think a panic is good. For example when a formatter encounters an interned pattern, it panics too.